### PR TITLE
change to the .format() function

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
change to the .format() function in horizons/ai/aiplayer/ files

### CODE REVIEW

File: horizons/ai/aiplayer/resourcemanager.py [hunk 1]
```diff
@@ -487,9 +488,11 @@ def get_total_export(self):
 	def __str__(self):
 		if not hasattr(self, "resource_id"):
 			return 'UninitializedSingleResourceManager'
-		result = 'Resource %d production %.5f/%.5f (%.5f low priority)' % (self.resource_id, self.available, self.total, self.low_priority)
+		result = 'Resource {:d} production {:.5f}/{:.5f] ({:.5f} low priority)'.format(
```
Comment (AndyMender) [inline horizons/ai/aiplayer/resourcemanager.py]: There is an error here. Look closely at the curly brace pairing:
```py
'Resource {:d} production {:.5f}/{:.5f] ({:.5f} low priority)'.format()
```

File: horizons/ai/aiplayer/villagebuilder.py [hunk 2]
```diff
@@ -854,4 +854,6 @@ def display(self):
 			renderer.addColored(tile._instance, *color)
 
 	def __str__(self):
-		return '%s VillageBuilder(%s)' % (self.settlement_manager, self.worldid if hasattr(self, 'worldid') else 'none')
+		return '{} VillageBuilder{})'.format(
```
Comment (AndyMender) [inline horizons/ai/aiplayer/villagebuilder.py]: You missed 1 brace here, should be:
```py
return '{} VillageBuilder({})'.format(
```

File: horizons/ai/aiplayer/resourcemanager.py (hunk 1 above)
Comment (MarkusHackspacher) [inline horizons/ai/aiplayer/resourcemanager.py]: Is solve.

**Comment by AndyMender** [comment]
Approved. For the future, I think it would be a good idea to give meaningful names to commits so that later we know what each of them did :).

**Pull request merged**